tests/phpunit/tests/xmlrpc/wp/newPost.php

```diff
@@ -240,11 +286,11 @@ function test_terms() {
 		$tag3 = wp_create_tag( 'tag3' );
 		$this->assertInternalType( 'array', $tag3 );
 
-		$post = array(
+		$post   = array(
 			'post_title' => 'Test',
-			'terms' => array(
-				'post_tag' => array( $tag2['term_id'], $tag3['term_id'] )
-			)
+			'terms'      => array(
+				'post_tag' => array( $tag2['term_id'], $tag3['term_id'] ),
+			),
 		);
 		$result = $this->myxmlrpcserver->wp_newPost( array( 1, 'editor', 'editor', $post ) );
```

Comment:
Should items in this array start on new lines since it has more than one item?

Reply:
It's not an associative array.